src/App.js

```diff
@@ -1,9 +1,65 @@
+/* eslint-disable import/extensions */
+/*
+reason for this is that I did try to change it several times
+and it gave me contradicting rules
+namely:
+- Missing file extension for "./components/Calculator"  import/extensions
+- Unexpected use of file extension "js" for "./components/home.js"   import/extensions
+- Missing file extension for "./components/home"
+*/
+import {
+  BrowserRouter, Routes, Route, Outlet, NavLink,
+} from 'react-router-dom';
 import './App.css';
 import Calculator from './components/calculator';
+import Home from './components/home';
+import Quote from './components/quote';
+
```

Comment:
- [ ] like the previous reviewer noted the error still exist. kindly change the part to link to the component by naming them exactly like they are... the home/quote should be changed to Home/Quote
![Screenshot (839)_LI](https://user-images.githubusercontent.com/65298048/221281662-26520b2f-517e-474f-819c-0179315c6689.jpg)

- [optional] please kindly centralize your calculator app to the middle of the page.